content/en/blog/_posts/2021-03-30-Kubernetes-Multi-Tenancy-Usage-Models.md

```diff
@@ -0,0 +1,80 @@
+
+### Clusters-as-a-Service
+
+With the *clusters-as-a-service* usage model, each tenant gets their own cluster.  This model allows tenants to have different versions of cluster-wide resources, such as CRDs.
+
+The tenant clusters may be provisioned using projects like [Cluster API (CAPI)](https://cluster-api.sigs.k8s.io/) where a management cluster is used to provision multiple workload clusters. A workload cluster is assigned to a tenant and tenants have full control over cluster resources. Note that in most enterprises a central platform team may be responsible for managing required add-on services such as security and monitoring services, and for providing cluster lifecycle management services such as patching and upgrades. A tenant administrator may be restricted from modifying the centrally managed services and other critical cluster information. 
+
+In a variation of the *clusters-as-a-service* model, the tenant cluster may be a **virtual cluster**. As with other forms of virtualization, users of a virtual cluster see no significant differences between a virtual cluster and other Kubernetes clusters. However, a virtual cluster shares worker node resources and some workload state independent control plane components, like the scheduler. Other workload aware control-plane components, like the API server, are created on a per-tenant basis and additional components are used to synchronize and manage state across virtual clusters and the underlying shared cluster. This provides each tenant their own virtual control-plane that can operate independently of other tenants.
+
+The [Virtual Cluster](https://github.com/kubernetes-sigs/multi-tenancy/tree/master/incubator/virtualcluster) project implements this model, where a supercluster is shared by multiple virtual clusters. The [Cluster API Nested](https://github.com/kubernetes-sigs/cluster-api-provider-nested) project is extending this work to conform to the CAPI model, allowing use of familiar API resources to provision and manage virtual clusters.
+
+## Summary
+
+A 2020 CNCF survey showed that production Kubernetes usage has increased by over 300% since 2016. As an increasing number of Kubernetes workloads move to production, organizations are looking for ways to share Kubernetes resources across teams for agility and cost savings. 
+
+The *namespaces-as-a-service* usage model allows sharing clusters and hence enables resource efficiencies. However, it requires proper security configurations and has limitations as all tenants share the same cluster-wide resources. 
+
+The *clusters-as-a-service* model addresses these limitations. However, having separate clusters per-app or per-team can be costly. *Virtual clusters* provide a way to share resources of a single Kubernetes cluster and also allow tenants to manage their own cluster-wide resources.
```

Comment:
"... addresses these limitations, **but with higher management and resource overhead.**" We shouldn't make it seem like there's no downsides to this one.